ui/src/app/cohort-review/participant.model.ts

```diff
@@ -14,7 +14,22 @@ export class Participant implements ParticipantCohortStatus {
     return Participant.formatStatusForText(this.status);
   }
 
+  get formattedGenderText() {
+    if (!this.gender) {
+      return;
+    }
+    return this.gender.charAt(0).toUpperCase() + this.gender.slice(1).toLowerCase();
+  }
+
+  get formattedDeceasedText() {
+    if (this.deceased === null) {
+      return;
+    }
+    return this.deceased ? 'Deceased' : 'Not Deceased';
```

Comment:
nit: maybe remove the if and use ternary operator?
```suggestion
    return (this.deceased !== null && this.deceased) ? 'Deceased' : 'Not Deceased';
```